pkg/cmd/repo/license/view/view.go

```diff
@@ -0,0 +1,99 @@
+package view
+
+import (
+	"errors"
+	"fmt"
+	"net/http"
+	"strings"
+
+	"github.com/MakeNowJust/heredoc"
+	"github.com/cli/cli/v2/api"
+	"github.com/cli/cli/v2/internal/gh"
+	"github.com/cli/cli/v2/pkg/cmdutil"
+	"github.com/cli/cli/v2/pkg/iostreams"
+	"github.com/spf13/cobra"
+)
+
+type ViewOptions struct {
+	IO         *iostreams.IOStreams
+	HTTPClient func() (*http.Client, error)
+	Config     func() (gh.Config, error)
+	License    string
+}
```

Comment:
**What do you think about a `--web` mode that brings users to the choosealicense.com website?**

For example, https://choosealicense.com/licenses/cern-ohl-s-2.0/

Reply:
I like it. I'll see if this is a low amount of work that we can squeeze into this PR.

Reply:
Done 👍